### ISSUE TITLE
Docs: Improve requirements page

### ISSUE THREAD

**Issue by marefr** [grafana/grafana#19101]
**What would you like to be added**:
In #18917 we've added a new requirements page that was a first rough version and we should improve that by adding some more information:
- Recommended hardware/database requirements

**Why is this needed**:
Help users understand what a recommended setup of Grafana would be like

**Comment by oddlittlebird** [comment]
@marefr , can we close this or do we still need to add info to the documentation?

**Comment by marefr** [comment]
@oddlittlebird no, still lacking a proper hardware recommendation given a "standard setup".

**Comment by oddlittlebird** [comment]
How can we get this? This should be an easy fix once we actually get the info.

**Comment by bergquist** [comment]
There are no correct answers to these questions. But we could give very rough estimates.

**Comment by achatterjee-grafana** [comment]
@marefr and @torkelo, where can we find the information to add. If you can let us know the SME to work with, we can update the docs and close the issue. Thanks for your help!

**Comment by achatterjee-grafana** [comment]
Hello @DeeGeeGit, can you let us know what information to add? If the docs are satisfactory, we can then close the issue. Thank you!

**Comment by GrafanaWriter** [comment]
I believe https://grafana.com/docs/grafana/next/setup-grafana/installation/#hardware-recommendations resolves this issue, but please re-open if more information is required.

**Comment by CptOrange16** [comment]
Hello, i still don't think the documentation is clear enough on the requirements

In the current [documentation](https://grafana.com/docs/grafana/next/setup-grafana/installation/#hardware-recommendations), it says that:
"Some features might require more memory or CPUs. Features require more resources include:
-   [Server side rendering of images](https://grafana.com/grafana/plugins/grafana-image-renderer#requirements)
-   [Alerting](https://grafana.com/docs/grafana/next/alerting/)
-   [Data source proxy](https://grafana.com/docs/grafana/next/developers/http_api/data_source/)"

Some features require more resources but in the page linked to those features there is no mention of any requirements. 
The exception being the Server side rendering of images page which mentions memory requirements.

 Basically we know they require more but there is no information on how much.
